sdk/Lusid.Sdk.Tests/tutorials/Instruments/EquityOptionExamples.cs

```diff
@@ -257,7 +256,7 @@ public void LifeCycleManagementForCashSettledEquityOption(ModelSelection.ModelEn
             var containsCashAfterUpsertion = valuationAfterUpsertingCashFlows
                 .Data
                 .Select(d => (string) d[TestDataUtilities.Luid])
-                .Any(luid => luid != $"CCY_{equityOption.DomCcy}");
+                .Any(luid => luid == $"CCY_{equityOption.DomCcy}");
```

Comment:
old code was bugged it seems

Reply:
good point, am ignoring tho since that’s just correcting a different test. would change it if it was my test